bellows/types/named.py

```diff
@@ -1405,6 +1406,14 @@ class EmberKeyStatus(basic.uint8_t, enum.Enum):
     TC_FAILED_TO_SEND_APP_KEYS = 0x0F
     TC_FAILED_TO_STORE_APP_KEY_REQUEST = 0x10
     TC_REJECTED_APP_KEY_REQUEST = 0x11
+    TC_FAILED_TO_GENERATE_NEW_KEY = 0x12
```

Comment:
I do not know what version they were added in, but I just looked through "SimplicityStudio_v4/developer/sdks/gecko_sdk_suite/v1.1/protocol/zigbee_5.10/stack/include/ember-types.h" and added what I found.